.github/workflows/ci.yml

```diff
@@ -25,36 +25,32 @@ jobs:
       with:
         fetch-depth: 0  # unshallow fetch for setuptools-scm
 
-    - name: Use MSBuild (Windows)
-      uses: microsoft/setup-msbuild@v1.0.2
-      if: matrix.os == 'windows-latest'
-
-    - uses: actions/setup-python@v1
-      name: Install Python
-      with:
-        python-version: '3.6'
-
-    - name: Install cibuildwheel
-      run: |
-        python -m pip install cibuildwheel==1.5.2
-
-    - name: Build wheel
-      run: |
-        python -m cibuildwheel --output-dir dist
+    - name: Build wheels
+      uses: pypa/cibuildwheel@v2.2.2
       env:
         # The packaged FreeType library is independent of the Python ABI so we only
         # need to build it once.
-        CIBW_BUILD: "cp36-*x86_64 cp36-*amd64"
+        CIBW_BUILD: "cp39-*"

```

Comment:
the reason it was built on a lower python minor version was to make sure that the tests pass on the minimum supported version. Arguably 3.6 is too low nowadays (EOL soon at end of year), how about setting it to 3.7?

Reply:
I set it back to 3.6 or 3.7, whatever you like, but arm64 needs 3.8 at a minimum, macos arm64 wheels aren't supported below that.

Reply:
Done.